resources/rules/post_identifier_merge/ice_to_bio/miscrna_gene_type_assertions.clj

```diff
@@ -0,0 +1,22 @@
+;; ----------------------------------------------
+;; --------- miscRNA Gene Type Assertion ---------
+;; -----------------------------------------------
+`{:name "miscrna-gene-type-assertion"
+  :description "This rule asserts miscRNA gene type"
+  ;; miscRNA maps to SO: (so:)
+  :head ((?/egKabobDna rdfs/subClassOf obo/SO_XXXXXX))
```

Comment:
@bill-baumgartner that sounds good. Do you want me to do that for the 'other' and ''unknown' rules as well?